examples/X11/X11.cpp

```diff
@@ -169,31 +171,49 @@ int main()
     XSetWindowAttributes attributes;
     attributes.background_pixel = BlackPixel(display, screen);
     attributes.event_mask       = KeyPressMask;
-    Window window = XCreateWindow(display, RootWindow(display, screen),
-                                  0, 0, 650, 330, 0,
+    Window window               = XCreateWindow(display,
+                                  RootWindow(display, screen),
```

Comment:
It's aligning the assignments with the previous assignments. I agree it looks a bit odd. clang-format-s `align` settings can be very nice but I tend to avoid them because of situations like this where clang-format gets a bit too eager to vertically align everything for no meaningful gain.